patches/snappy/snappy-1.1.9_remove-no-rtti.patch

```diff
@@ -0,0 +1,35 @@
+From 516fdcca6606502e2d562d20c01b225c8d066739 Mon Sep 17 00:00:00 2001
```

Comment:
This file does not match our usual patch naming convention.

Reply:
The patch was taken as-is from gentoo, so I kept the name intact as well.

Reply:
The issue is that xbstrap sorts patches by their file name, so if we're missing the `0001-` prefix, they'll be applied in "random" orders.